src/App.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import './App.css';
+import Tab from './components/Tab';
+
+class App extends React.Component {
+  state = {
+    tabs: [
```

Comment:
when using stubs, create a different file under `constants` folder and import it from there.
Otherwise your components will become bloated with unnecessary data.

Another approach will be, to have a constants.js specifically for this component.
You will have a folder for tabs inside component, where all your tab components and constants will sit together.